CMakeLists.txt

```diff
@@ -192,11 +202,16 @@ set(headers
     src/xml.h
     src/dict.h)
 
+if(WIN32)
+    # use a Windows getopt_long implementation
+    configure_file(${PROJECT_SOURCE_DIR}/src/getopt.h.in ${PROJECT_BINARY_DIR}/src/getopt.h)
+    set(libsrc ${libsrc} src/getopt.c)
+    set(headers ${headers} src/getopt.h)
+endif()
+
 # link compat
 use_compat()
 
-set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
-
```

Comment:
Without this, CFFI python bindings do not build when `-DENABLE_STATIC=1`. I recently enabled -fPIC for all compiled code in this commit:

https://github.com/CESNET/libyang/commit/6e6475d8d5ad175c78d523b73a0304ddc59e116b#diff-af3b638bc2a3e6c650974192a53c7291R198

Could you please take this into account? Maybe win32 does not support static build at all and in that case, you should fail fast accordingly.

Reply:
I restored it.